frontends/open-discussions/package.json

```diff
@@ -31,8 +36,18 @@
     "@mitodl/ckeditor-custom-build": "0.0.4",
     "@mitodl/course-search-utils": "^2.3.0",
     "@mitodl/mdl-react-components": "0.0.3",
+    "@mui/icons-material": "^5.8.4",
+    "@mui/material": "^5.9.0",
     "@reduxjs/toolkit": "1.4.0",
+    "@rmwc/checkbox": "^5.7.2",
+    "@rmwc/dialog": "^5.7.2",
+    "@rmwc/drawer": "^5.7.2",
+    "@rmwc/menu": "^5.7.2",
+    "@rmwc/snackbar": "^5.7.2",
+    "@rmwc/theme": "^5.7.2",
```

Comment:
Previously we were getting these (and `material-components-web`) from `rmwc`; since `rmwc` was never being used directly, installing these is equivalent, and makes `import/no-extraneous-dependencies` happy.